fetch.bs

```diff
@@ -1162,6 +1162,10 @@ Unless stated otherwise, it is "<code>basic</code>".
 <p>A <a for=/>request</a> has an associated <dfn export for=request id=done-flag>done flag</dfn>.
 Unless stated otherwise, it is unset.
 
+<p>A <a for=/>request</a> has an associated
+<dfn export for=request id=concept-request-reload-navigation-flag>reload-navigation flag</dfn>.
+Unless stated otherwise, it is unset.
```

Comment:
Let's add a note here that this is for the exclusive use of HTML's navigate algorithm.

Reply:
Done.